### DISCUSSION TITLE
how can I call groups of light ?

### DISCUSSION THREAD

**Discussion by madirshah** [jseidl/magic-areas#171] [Q&A]
Hi,

thanks for this great work!

I am trying to use the automatically generated light group in the area in my yaml to define a button for the light group. (on/off)
I have no idea how to do this? please help.

**Comment by jseidl** [comment]
Can you elaborate? Light groups will be a toggle in the UI that you can turn on and off.

**Comment by jseidl** [comment]
Any updates on this?
